### DISCUSSION TITLE
Solana social media

### DISCUSSION THREAD

**Discussion by Manuel-yang** [creatorsand/.github#9] [Ideas]
Roseite 全数据上链社交平台
问题：
在web3 socialFi有许多次大胆的尝试和新颖的想法，但大多数产品都未能实现内容产出与收益的良性循环。并且由于web3去中心化的特性，socialFi中充斥着大量无用的垃圾消息。

背景：
相比于ETH高昂的手续费以及数据存储费用，Solnana更加便宜且高效，故此本项目应用将在solana链上搭建并实现。 与过往的web3 socialFi产品所不同的是，Roseite将会把账户Nft化，也就是以nft作为社交账户（得益于solana独特的pda模型），而其账户下的内容将会通过PDA存储至链上以实现全数据上链。

优势：
1. 使得账户更加去中心化：谁拥有此Nft账户谁将掌握该账户的所有权（包括账户下的post所有权以及其他所有收益）
2. 更好的激励机制: 用户可以主动向高质量post pda直接打赏，而其post 所属nft的持有者可以直接从post pda中领取收益以达到高质量内容输出与收益的良好循环
3. 更高的Nft价值： Nft的价值将不再由炒作体现，Nft持有者通过输出优质内容吸引粉丝且创造出更多可收益的post，提高Nft账户流量。同时Nft可上架主流Nft marketplace如：Magic edgn等进行售卖
4. 优秀的协议： 得益于Nft账户的设计，我们将在Solana链上实现类ERC-6551的协议使得该Nft账户之下同时能拥有其他子Nft。在主Nft进行归属权转移时，子Nft同样也会进行转移。但同时子Nft也同样可以进行单独转移
5. 更好的内容筛选机制： 得益于Solana独特的模型机制，我们希望能通过控制Post pda租金的方式来达到优质内容的输出以及无用内容的减少，来保持Roseite的健康。

潜力：
- 由于web3的透明性，用户画像能更好的实现
- 使用Solana中Lookup table特性，用户最高能在链上存储高达 255 * 3MB = 756MB的内容
- 根据solana独特的模型扩展类ERC-6551协议使得玩法更多元

**Comment by Cobb9527** [comment]
蹲一个后续~
